src/test/java/com/commercetools/sync/taxcategories/TaxCategorySyncOptionsBuilderTest.java

```diff
@@ -0,0 +1,138 @@
+package com.commercetools.sync.taxcategories;
+
+import com.commercetools.sync.commons.utils.TriFunction;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.taxcategories.TaxCategory;
+import io.sphere.sdk.taxcategories.TaxCategoryDraft;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.mock;
+
+class TaxCategorySyncOptionsBuilderTest {
```

Comment:
in general, I think we should reconsider the usage of the `.as` for assertion descriptions

Reply:
At first it sounded like a good idea but after giving it some thoughts I think using .as is a bit redundant.
When test ends in a failure we have information what was wrong - test case has to be reviewed to find out what really happened. I will remove those _.as_ descriptions.